diff -urN wmappl-0.61.orig/args.h wmappl-0.61/args.h
--- wmappl-0.61.orig/args.h	2002-03-21 09:51:14.000000000 +0100
+++ wmappl-0.61/args.h	2004-02-05 20:47:40.000000000 +0100
@@ -28,7 +28,7 @@
 
 /* defaults macros */
 #define WMAPPLRC ".wmapplrc"
-#define ICONPATH "/usr/share/icons/wmappl:/usr/share/pixmaps/mini"
+#define ICONPATH "/usr/share/wmappl/icons:/usr/share/pixmaps/mini"
 
 
 /* MACRO defining the path separator */
diff -urN wmappl-0.61.orig/sample.wmapplrc wmappl-0.61/sample.wmapplrc
--- wmappl-0.61.orig/sample.wmapplrc	2002-03-21 09:39:45.000000000 +0100
+++ wmappl-0.61/sample.wmapplrc	2004-02-05 21:13:01.000000000 +0100
@@ -57,7 +57,7 @@
 
 [wprefs]
 icon = gnustep.xpm
-command = /usr/X11R6/lib/GNUstep/Apps/WPrefs.app/WPrefs &
+command = /usr/lib/GNUstep/Apps/WPrefs.app/WPrefs &
 
 [greenterm]
 icon = xterm.xpm
